src/DetailsView/components/reflow-assessment-view.tsx

```diff
@@ -39,10 +47,22 @@ export class ReflowAssessmentView extends React.Component<ReflowAssessmentViewPr
         return null;
     }
 
+    public componentDidMount(): void {
+        this.props.deps.assessmentViewUpdateHandler.onMount(this.props);
+    }
+
+    public componentDidUpdate(prevProps: ReflowAssessmentViewProps): void {
+        this.props.deps.assessmentViewUpdateHandler.update(prevProps, this.props);
+    }
+
+    public componentWillUnmount(): void {
+        this.props.deps.assessmentViewUpdateHandler.onUnmount(this.props);
+    }
+
     private renderTargetChangeDialog(): JSX.Element {
         return (
             <TargetChangeDialog
-                deps={this.props.deps}
+                deps={this.props.deps as TargetChangeDialogDeps}
```

Comment:
Out of curiosity, why did you have to specify a type for the deps? If you look at `AssessmentView` I'm not sure you have to specify the deps type as `TargetChangeDialog,` especially since `ReflowAssessmentViewDeps` includes `TargetChangeDialogDeps`

Reply:
Took this out.